### DISCUSSION TITLE
Correct Labelling Procedure for Multi-Instance Bounding Boxes

### DISCUSSION THREAD

**Discussion by CCEKael** [ultralytics/ultralytics#3038] [Q&A]
Hey there,

I'm conducting a computer vision labelling project and have read conflicting opinions on the need to draw bounding boxes around all instances of a product if multiple appear in an image. Or whether simply drawing a bounding box around the best instance will suffice.

For example some say this is necessary:
![img_004 (2)](https://github.com/ultralytics/ultralytics/assets/135681101/93911d70-b6fa-4fc3-8ba9-8446942e7623)

While others say with newer models such as yolo 8 this will do:
![img_004 (2)](https://github.com/ultralytics/ultralytics/assets/135681101/ecb3432c-78ad-45bc-be98-6e5b734cc3e8)

Our preference would be to do the latter. Would there be any downsides with yolo 8 and this approach?

Thanks.

**Comment by glenn-jocher** [comment]
@CCEKael hello!

For the best performance with YOLOv8, it's recommended to annotate all instances of the objects of interest in your images. This allows the model to learn from the full context and variety present in your dataset, which typically results in better detection accuracy, especially in scenarios where multiple instances of the same object are common.

If you only label the "best" instance, the model may not learn to recognize multiple objects in close proximity or partially occluded objects. This could lead to missed detections or poor generalization to images with multiple instances.

So, to maximize your model's effectiveness, it's advisable to label all instances. Happy labeling! 😊📦
